scripts/release/generate_history_notes.py

```diff
@@ -26,8 +26,38 @@
 history_line_breaker = '==============='
 history_notes = {}
 
+# key is lower case and removed spaces of a component
+# value is the recommended format of a component
+customized_dict = {
+    "privatedns": "Private DNS",
+    "appconfig": "App Config",
+}
+
+# This dict will be filled with all historical compenents.
+# key is lower case and removed spaces of a component
+# value is the recommended format of a component, when there're multiple formats,
+# the one with spaces will be picked, i.e. pick 'Key Vault' over 'KeyVault'.
+# If the key also exists in customized_dict, the value will be overwritten with customized_dict[key].
+component_dict = {}
+
+def get_component_dict():
+    with open('src/azure-cli/HISTORY.rst', 'r', encoding="utf-8") as history_file:
+        for line in history_file:
+            result = re.search(r'^\*\*(.*)\*\*$', line)
+            if result is not None:
+                comp = result.group(1)
+                key = comp.lower().replace(' ', '')
+                if key in customized_dict:
+                    component_dict[key] = customized_dict[key]
+                elif key in component_dict:
+                    if ' ' not in component_dict[key] and ' ' in comp:
+                        component_dict[key] = comp
+                else:
+                    component_dict[key] = comp
+    print(component_dict)
```

Comment:
logger?

Reply:
It's a script used in pipeline. print should be OK, it's logged by ADO.